haystack/nodes/reader/farm.py

```diff
@@ -887,6 +891,7 @@ def eval_on_file(
         :param device: The device on which the tensors should be processed.
                Choose from torch.device("cpu") and torch.device("cuda") (or simply "cpu" or "cuda")
                or use the Reader's device by default.
+        :param calibrate_conf_scores: Whether to calibrate the temperature for temperature scaling of the confidence scores
```

Comment:
Could we just say "Whether to calibrate the temperature for scaling of the confidence scores."?